src/controller/python/chip/ChipDeviceCtrl.py

```diff
@@ -1427,3 +1269,203 @@ def _InitLib(self):
 
             self._dmLib.pychip_DeviceController_GetLogFilter = [None]
             self._dmLib.pychip_DeviceController_GetLogFilter = c_uint8
+
+            self._dmLib.pychip_OpCreds_AllocateController.argtypes = [c_void_p, POINTER(
+                c_void_p), c_uint64, c_uint64, c_uint16, c_char_p, c_bool, c_bool, POINTER(c_uint32), c_uint32]
+            self._dmLib.pychip_OpCreds_AllocateController.restype = PyChipError
+
+            self._dmLib.pychip_OpCreds_AllocateControllerForCustomCommissioningFlow.argtypes = [
+                POINTER(c_void_p), c_char_p, c_uint32, c_char_p, c_uint32, c_char_p, c_uint32, c_char_p, c_uint32, c_uint16, c_bool]
+            self._dmLib.pychip_OpCreds_AllocateControllerForCustomCommissioningFlow.restype = PyChipError
```

Comment:
Suggest replacing "CustomCommissioningFlow" everywhere with "PythonCommissioningFLow" --> it avoids confusion with "custom flows" in spec

Reply:
Done